app/errors.py

```diff
@@ -0,0 +1,16 @@
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    def __str__(self) -> str:
+        return "Friends should buy masks"
```

Comment:
and there

Reply:
about this - agree

Reply:
If I delete it - I will have "AssertionError: NotWearingMaskError should be raised with a message" 
For sure, I can add message to the visit_cafe function, but for me str method looks better